tests/containers/base_image_test.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import annotations
+
+import logging
+import pathlib
+import tempfile
+from typing import TYPE_CHECKING
+
+import testcontainers.core.container
+import testcontainers.core.waiting_utils
+
+from tests.containers import docker_utils
+
+logging.basicConfig(level=logging.DEBUG)
+LOGGER = logging.getLogger(__name__)
+
+if TYPE_CHECKING:
+    import pytest_subtests
+
+
+class TestBaseImage:
```

Comment:
@jstourac wondered how to structure the tests when more are to be added, like when the version tests are rewritten from makefile to here